docusaurus/docs/guides/deployments/20-docker/10-controller.mdx

```diff
@@ -0,0 +1,8 @@
+---
+title: Deploy the Controller with Docker
+sidebar_label: Controller
+---
+
+import DockerControllerReadme from '/_remotes/ziti-cmd/dist/docker-images/ziti-controller/README.md';
```

Comment:
i'm all for reusing these here but can we please remove the codeblock hints? the rendering in docusaurus is still lackluster and we don't have the time to polish it right now

Reply:
https://github.com/openziti/ziti/pull/1987